Update companion images description on Docker Hub (#753)
yuzutech/kroki: DOCKERHUB-COMPANION.md

> # Kroki Companion Image

Reviewer: I think we should say "Kroki Companion Server".
In my opinion "Image" is redundant since we are on hub.docker.com

> Image to use together with this server: [yuzutech/kroki](https://hub.docker.com/r/yuzutech/kroki)

Reviewer: ```suggestion
Provides additional diagram libraries to [yuzutech/kroki](https://hub.docker.com/r/yuzutech/kroki).
```

> GitHub repository: [kroki](https://github.com/yuzutech/kroki)

Reviewer: I think it would be better to link to the user documentation no?

https://docs.kroki.io/kroki/setup/install/#_images

```suggestion
Read the [Kroki installation](https://docs.kroki.io/kroki/setup/install/#_images) to learn how to start Kroki companion containers.
```

Reviewer: Having said that it would be useful to add the GitHub repository but I don't know where to configure it:

![image](https://user-images.githubusercontent.com/333276/118892910-ea2f3400-b901-11eb-84a3-8331c81fe034.png)

![image](https://user-images.githubusercontent.com/333276/118892961-fc10d700-b901-11eb-9a64-4b3d393e352e.png)

😞

Reviewer: That's what I suspected, thanks for the investigation.

>  when you use automated builds in Dockerhub that's not an option I presume.

Indeed, not an option for us :(

## Usage

Use this images with:

> * [Docker-compose](https://docs.kroki.io/kroki/setup/install/#_using_docker_compose)

Reviewer: ```suggestion
* [docker-compose](https://docs.kroki.io/kroki/setup/install/#_using_docker_compose)
```

* [Kubernetes](https://docs.kroki.io/kroki/setup/install/#_using_kubernetes)

> For installation and configuration see [Kroki docs](https://docs.kroki.io/)

Reviewer: ```suggestion
To learn more about Kroki, read the [documentation](https://docs.kroki.io/).
```
